src/lib/buildComment.ts

```diff
@@ -22,7 +22,12 @@ function isSomKindOfJsxAtLine(
 
   const isJsxStartOpeningElement =
     targetNode?.getPreviousSibling()?.getKind() ===
-    ts.SyntaxKind.OpenParenToken;
+      ts.SyntaxKind.OpenParenToken ||
+    // This can happen when the error is on the opening JSX tag inside
+    // a JSX expression when a block of JSX is being passed as a prop.
```

Comment:
**suggestion:** It is hard to visualize this from the comment, code, and test cases; could you give a before/after example in this PR?

Reply:
It was adding something like:
```
<Foo
{/*
  // ts-expect-error TS2769 */}
  bar={baz}
/>
```

I'll include an example in comment before landing.